apparmor.d/abstractions/bus/org.freedesktop.PackageKit

```diff
@@ -2,6 +2,41 @@
 # Copyright (C) 2023 Alexandre Pujol <alexandre@pujol.io>
 # SPDX-License-Identifier: GPL-2.0-only
 
+  dbus receive bus=system path=/@{int}@{int}@{int}@{int}@{int}_edcedacc
+       interface=org.freedesktop.PackageKit.Transaction
+       member=GetUpdates
+       peer=(name=:*),
```

Comment:
You **must** specify the peer label: this is one of the value added by this project: as we have profile for everything, we can restrict the dbus further setting the peer label (the profile name). 

*(same everywhere you did not mention peer label)*